src/resdk/tables/variant.py

```diff
@@ -0,0 +1,157 @@
+""".. Ignore pydocstyle D400.
+
+=============
+VariantTables
+=============
+
+.. autoclass:: VariantTables
+    :members:
+
+"""
+import re
+from functools import lru_cache
+
+import pandas as pd
+
+from .base import BaseTables
+
+
+class VariantTables(BaseTables):
+    """A helper class to fetch collection's variant, qc and meta data.
+
+    This class enables fetching given collection's data and returning it
+    as tables which have samples in rows and variants / qc / metadata
+    in columns.
+
+    A simple example:
+
+    .. code-block:: python
+
+        # Get Collection object
+        collection = res.collection.get("collection-slug")
+
+        # Fetch collection variant data
+        tables = VariantTables(collection)
+        var = tables.variants
+
+    """
+
+    process_type = "data:mutationstable"
+    VARIANTS = "variants"
+    DEPTH = "depth"
+    COVERAGE = "coverage"
```

Comment:
Reporting coverage is OK, but reporting base frequencies would offer even more insight about the variant. If we indeed report base frequencies then maybe coverage report can be removed.